packages/adapter-prisma/src/index.ts

```diff
@@ -29,13 +29,22 @@ export function PrismaAdapter(p: PrismaClient): Adapter {
     },
     createSession: (data) => p.session.create({ data }),
     updateSession: (data) =>
```

Comment:
This is the only place we call `updateSession` right now:

https://github.com/nextauthjs/next-auth/blob/05803ba36cccd069905d50db25c4459ab77c9f69/packages/next-auth/src/core/routes/session.ts#L122

so we never pass it the `id`, but I don't think it would be a problem anyway, as we do the lookup with `sessionToken`.